docs-2.0/9.about-license/4.generate-and-load-license-key.md

```diff
@@ -0,0 +1,53 @@
+# Generate and load license keys
+
+After purchasing a license, you need to generate a [license key](2.license-management-suite/2.license-center.md) and then load it into the [LM](2.license-management-suite/3.license-manager.md). After that, NebulaGraph and its associated software check the validity of the license key through LM at startup. If the license key is valid, the NebulaGraph database and associated software can be used normally. This article introduces how to generate and load a license key.
+
+## Preparations
+
+- [You have purchased a license](3.purchase-license.md)
+- [You have registered an LC account and logged in](2.license-management-suite/2.license-center.md)
+- [You have installed LM](2.license-management-suite/3.license-manager.md)
+
+## Generate a license key
+
+1. [Go to LC](2.license-management-suite/2.license-center.md).
+2. In the **LICENSE LIST** section, click the target license card.
+3. On the **License Info** page, find the **Basic Info** section, and click **Bind License Manager ID**.
+4. Bind the ID of your LM. For details, see [Bind LMID](2.license-management-suite/2.license-center.md).
+
+  !!! note 
+
+        You can choose a license key type based on your LM deployment environment. 
+
+      - If your LM is deployed on a public network, both **Online** and **Offline** modes can be selected. **Online** mode is recommended, which generates an online license key.
+      - If your LM is deployed on a private network, select the **Offline** mode, which generates an offline license key.
```

Comment:
Here the description is not accurate, it doesn't say that LM is deployed on a public network or not, it says if the LM is can be access to the public network, please note the difference